src/joystick/windows/SDL_rawinputjoystick.c

```diff
@@ -734,7 +739,7 @@ RAWINPUT_AddDevice(HANDLE hDevice)
         device->guid.data[15] = 0;
     }
 
-    hFile = CreateFileA(dev_name, GENERIC_READ, FILE_SHARE_READ|FILE_SHARE_WRITE, NULL, OPEN_EXISTING, 0, NULL);
+    hFile = CreateFileA(dev_name, 0, FILE_SHARE_READ | FILE_SHARE_WRITE, NULL, OPEN_EXISTING, 0, NULL);
```

Comment:
What's the reason for this change?

Reply:
We don’t need read access to read device info from handle. So just don’t request it.

Reply:
Yep, that looks accurate. The underlying HID IOCTLs are all declared as FILE_ANY_ACCESS.